app/features/build/build_controller.rb

```diff
@@ -51,5 +51,24 @@ class BuildController < AuthenticatedControllerBase
       }
       erb(:build, locals: locals, layout: FastlaneCI.default_layout)
     end
+
+    # convert .log files that include the color information as ANSI code
+    # back to HTML code that can be rendered by the user's browser
+    # We probably want to re-visit this in the future, but for now it's good
+    def convert_ansi_to_html(data)
+      {
```

Comment:
maybe instantiate this once as a class variable?

Reply:
ok